steps/embrace-symbol-upload/1.0.0/step.yml

```diff
@@ -0,0 +1,38 @@
+title: Embrace iOS Symbol Upload
+summary: |
+  Upload iOS debug symbols to Embrace through Bitrise CI integration
+description: |
+  Upload iOS debug symbols to Embrace through Bitrise CI. This will make your integration with Embrace more efficient. Environment Variables are supported via the upload tool.
+
+  Env Vars:
+  ---------
+  EMBRACE_APP: Your Embrace app idenifier
+  EMBRACE_API_TOKEN: Your Embrace upload API token
+website: https://github.com/embrace-io/bitrise-step-embrace-symbol-upload
+source_code_url: https://github.com/embrace-io/bitrise-step-embrace-symbol-upload
+support_url: https://github.com/embrace-io/bitrise-step-embrace-symbol-upload/issues
+published_at: 2023-10-05T14:49:30.418918-04:00
+source:
+  git: https://github.com/embrace-io/bitrise-symbol-upload.git
+  commit: f24fc1c9e96311333c2deab2ae69f3ad4b13409b
+project_type_tags:
+- ios
+type_tags:
+- artifact-info
+toolkit:
+  bash:
+    entry_file: step.sh
```

Comment:
I had a look at what this script does, and it's not clear for me where the CLI binary is coming from. Is it expected that the users of the step perform some other thing first? If so, it's probably better to highlight in the step description and handle the missing binary gracefully with a nice error message.

Reply:
At the moment the binaries are checked in with the step definition. Currently the tool is only shipped as part of our Cocoapod, but we're in the process of defining more distribution mechanisms, expanding the platforms supported by the CLI, and adding more capabilities to it in general. Eventually the intent is to add a separate step which installs the CLI.

Reply:
Makes sense, thank you